### ISSUE TITLE
Firewall needs to access MMGAT API

### ISSUE THREAD

**Issue by boris-ning-usds** [CDCgov/data-exchange-hl7#140]
Neelima and Cosmin is looking to gain access to https://mmgat.services.cdc.gov for MMGAT specifications.

Our Azure functions hitting the endpoint above is blocked by a firewall; and it requires us to go through a firewall process 

- [x] Gain access to file a CDC firewall ticket via an ISSO.
- [x] Request firewall changes between our subnet to the service's IP address range, C73511
- [ ] Document CDC Firewall process as part of the infrastructure needs (in the SOP stub)

**Comment by boris-ning-usds** [comment]
Firewall ticket submitted: [C73461](http://ithelpdesk.cdc.gov/webtier-9.34/index.do?ctx=docEngine&file=cm3r&query=number%3D%22C73461%22&action=&title=Change%20C73461&queryHash=d4aec3e0) to try to gain access to the MMG API service.

**Comment by boris-ning-usds** [comment]
I found a problem with the destination IP range provided by the cloud team for the MMG API service; it was missing an IP address. I tried asking the firewall team to move my ticket back, but that didn't work.

I created a brand new ticket and [C73511](http://ithelpdesk.cdc.gov/webtier-9.34/index.do?ctx=docEngine&file=cm3r&query=number%3D%22C73461%22&action=&title=Change%20C73511&queryHash=d4aec3e0) and submitted it over to our ISSO for review.

**Comment by rmharrison** [comment]
NB: Reviews on Tue and Thu. C73511 submitted on Thu, so it should be reviewed today.

**Comment by rmharrison** [comment]
Status, end of Sprint 22 Sep - 5 Oct: Waiting for Firewall team

**Comment by boris-ning-usds** [comment]
Firewall team indicated this has been completed. We need to test and verify.

**Comment by boris-ning-usds** [comment]
@dtx0111 , since you've been using the MMGAT API for your function app - this ticket is now completed.
Are you seeing any issues with access to MMGAT API?

**Comment by boris-ning-usds** [comment]
This was verified by @dtx0111 and she's been using it.
